packages/cms-lib/ignoreRules.js

```diff
@@ -46,7 +46,8 @@ function loadIgnoreConfig() {
 function shouldIgnoreFile(file, cwd) {
   loadIgnoreConfig();
   const relativeTo = configPath || cwd;
-  return ignoreRules.ignores(path.relative(relativeTo, file));
+  const relativePath = path.relative(relativeTo, file);
+  return !!relativePath && ignoreRules.ignores(relativePath);
```

Comment:
I think it would be best to start with a test case here because this fix suggests that there is something off elsewhere.

Reply:
I guess that was hidden in the chaos with the other commit. I actually added a test for this, as you might be able to see now that I removed the commit again

Reply:
While this change is a good one, I think that the issue is actually with how we're using `shouldIgnoreFile` in the `chokidar.watch()` call in `lib/watch.js`.

Reply:
Yes the fix smells like there should be something cleaned up, I'm unsure what though. The emptystring seems to be returned when `file == cwd`. I'd expect shouldIgnoreFile in general to treat such input gracefully…

Reply:
Yeah, I agree. If you make the small adjustment to the test, I'll get this merged. We can improve `lib/watch.js` separately.

Reply:
Hi gcorne, As I said there, that adjustment won't do, as you suggest.